docs/api/breaking-changes.md

```diff
@@ -6,6 +6,59 @@ Breaking changes will be documented here, and deprecation warnings added to JS c
 
 The `FIXME` string is used in code comments to denote things that should be fixed for future releases. See https://github.com/electron/electron/search?q=fixme
 
+## Planned Breaking API Changes (8.0)
```

Comment:
Are we targeting this for 8 or 9?  The PR says 9.

Reply:
This PR is targeted to 8. When 9 rolls around, the behavior will change again (the deprecated things will start throwing an exception instead of being serialized with the old algorithm).

Reply:
@nornagon doesn't that mean it will be `Planned Breaking API Changes (9.0)` in the doc?

Reply:
Ehh... nevermind I misread this.